### PR TITLE
Add node_jobs command

### PR DESCRIPTION
**Pull request by zmc**
Lists the last 10 jobs run on a given node

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

File: paddles/commands/node_jobs.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    def run(self, args):
+        super(NodeJobsCommand, self).run(args)
+        self.load_app()
+        models.start()
+        node_name = args.node
+        node_obj = Node.query.filter(Node.name.startswith(node_name)).one()
+        jobs = Job.query.filter(
+            Job.target_nodes.contains(node_obj)).filter(
+                ~Job.updated.is_(None)).order_by(
+                    Job.updated.desc()).limit(10)
```
Comment (andrewschoen) [inline paddles/commands/node_jobs.py]: Can 10 here be an argument to the command?

Comment (zmc) [inline paddles/commands/node_jobs.py]: I've never gotten optional arguments to work for these commands. If I could, I'd certainly make it an optional arg. As it stands now, though, I've never needed to see more than a few jobs (honestly, one job) from this output so making it required would mostly just be a pain.

Comment (andrewschoen) [inline paddles/commands/node_jobs.py]: Does the 'optional' kwarg not work when defining the arguments?

See the first example here: http://pecan.readthedocs.org/en/latest/pecan_commands.html

Comment (zmc) [inline paddles/commands/node_jobs.py]: Nope, that doesn't work:
``
TypeError: __init__() got an unexpected keyword argument 'optional'
``

However, I did just find the right way to do it. I'll re-push.

**Comment by zmc** [comment]
The count flag is optional now.

**Pull request merged**